pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -853,6 +853,33 @@ func Convert_v1_Rng_To_api_Rng(_ *v1.Rng, rng *api.Rng, c *ConverterContext) err
 	return nil
 }
 
+func Convert_v1_Usbredir_To_api_Usbredir(vmi *v1.VirtualMachineInstance, domainDevices *api.Devices, c *ConverterContext) (bool, error) {
```

Comment:
`c *ConverterContext` is an unused parameter

Reply:
That's true, I think I was following the other implementations. I'll double check and remove.